source_control/gitlab_project.py

```diff
@@ -0,0 +1,395 @@
+            owner = result['namespace']['name']
+            if owner == project_owner:
+                return True
+        return False
+
+    def existsGroup(self, group_name):
+        if group_name is not None:
+            # Find the group, if group not exists we try for user
+            for group in self._gitlab.getall(self._gitlab.getgroups):
+                if group['name'] == group_name:
+                    return True
+
+            user_name = group_name
+            user_data = self._gitlab.getusers(search=user_name)
+            for data in user_data:
+                if 'id' in user_data:
+                    return True
+        return False
+
+    def getGroupId(self, group_name):
+        if group_name is not None:
+            # Find the group, if group not exists we try for user
+            for group in self._gitlab.getall(self._gitlab.getgroups):
+                if group['name'] == group_name:
+                    return group['id']
+
+    def getProjectId(self, group_name, project_name):
+        if self.existsGroup(group_name):
+            project_owner = group_name
+        else:
+            project_owner = self._gitlab.currentuser()['username']
+
+        search_results = self._gitlab.searchproject(search=project_name)
+        for result in search_results:
+            owner = result['namespace']['name']
+            if owner == project_owner:
+                return result['id']
+
+    def getUserId(self, user_name):
+        user_data = self._gitlab.getusers(search=user_name)
+
+        for data in user_data:
+            if 'id' in data:
+                return data['id']
+        return self._gitlab.currentuser()['id']
+
+    def to_bool(self, value):
+        if value:
+            return 1
+        else:
+            return 0
+
+    def updateProject(self, group_name, arguments):
+        project_changed = False
+        project_name = arguments['name']
+        project_id = self.getProjectId(group_name, project_name)
+        project_data = self._gitlab.getproject(project_id=project_id)
+
+        for arg_key, arg_value in arguments.items():
+            project_data_value = project_data[arg_key]
+
+            if isinstance(project_data_value, bool) or project_data_value is None:
+                to_bool = self.to_bool(project_data_value)
+                if to_bool != arg_value:
+                    project_changed = True
+                    continue
+            else:
+                if project_data_value != arg_value:
+                    project_changed = True
+
+        if project_changed:
+            if self._module.check_mode:
+                self._module.exit_json(changed=True)
+            return self._gitlab.editproject(project_id=project_id, **arguments)
+        else:
+            return False
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            server_url=dict(required=True),
+            verify_ssl=dict(required=False, default=True, choices=BOOLEANS),
```

Comment:
in other modules the name for this param is `validate_certs` e.g. in http://docs.ansible.com/ansible/uri_module.html if you don't like to update the param name here, just add a alias e.g. `aliases=['validate_certs']` but change the docs in the way that `verify_ssl` is the alias and `validate_certs` is the param name.